x/oracle/keeper/common/types.go

```diff
@@ -153,7 +153,7 @@ func NewSet[T comparable](length int) *Set[T] {
 }
 
 func ExceedsThreshold(power *big.Int, totalPower *big.Int) bool {
-	return new(big.Int).Mul(power, big.NewInt(ThresholdB)).Cmp(new(big.Int).Mul(totalPower, big.NewInt(ThresholdA))) > 0
+	return new(big.Int).Mul(power, big.NewInt(int64(ThresholdB))).Cmp(new(big.Int).Mul(totalPower, big.NewInt(int64(ThresholdA)))) > 0
```

Comment:
can you explain why do this int64 convert ? 
no loss of precision when performing large integer calculations ?

Reply:
If it is from `int32` to `int64` it is safe right?

Reply:
big.NewInt need int64 as input, and the convert is safe